drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -79,5 +79,12 @@ config ETH_STM32_HAL_MAC5
 	  This is the byte 5 of the MAC address.
 endif
 
+config ETH_STM32_HAL_MII
+	bool "Use MII interface"
+	depends on ETH_STM32_HAL
+	default n
```

Comment:
Kconfig symbols already default to n.
Please remove this line

Reply:
fixed